### PR TITLE
fix simulator compiling

### PR DESCRIPTION
**Pull request by PPpro**

### CODE REVIEW

**Comment by PPpro** [comment]
@PatriceJiang @minggo pls review

**Review by PatriceJiang — APPROVED** [review]

File: tools/simulator/frameworks/runtime-src/CMakeLists.txt [hunk 1]
```diff
@@ -31,7 +31,7 @@ if(CMAKE_OSX_SYSROOT STREQUAL "iphonesimulator")
 else()
     set(TARGET_IOS_VERSION "8.0")
 endif()
-set(TARGET_OSX_VERSION "10.10")
```
Comment (minggo) [inline tools/simulator/frameworks/runtime-src/CMakeLists.txt:34]: Why need to change it?

Comment (PPpro) [reply tools/simulator/frameworks/runtime-src/CMakeLists.txt:34]: can't  use std:get  method when `TARGET_OSX_VERSION` is 10.10

Comment (PatriceJiang) [reply tools/simulator/frameworks/runtime-src/CMakeLists.txt:34]: 编译 tbb 用到 `std::get`,  要求有 macos 10.14+.    引擎的最低版本也是 10.14.

Comment (minggo) [reply tools/simulator/frameworks/runtime-src/CMakeLists.txt:34]: 所以还是得去除对 tbb 对依赖

**Pull request merged**